client/src/components/History/services.js

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { prependPath } from "utils/redirect";
+import { rethrowSimple } from "utils/simple-error";
+
+export class Services {
+    async getPublishedHistories(options = {}) {
```

Comment:
This can be exported directly as a function without the `Services` class wrapper. Since the class is stateless.